src/modelframe.jl

```diff
@@ -229,19 +229,19 @@ function coefnames(mf::ModelFrame)
     terms = droprandomeffects(dropresponse!(mf.terms))
 
     ## strategy mirrors ModelMatrx constructor:
-    eterm_names = @compat Dict{Tuple{Symbol,Bool}, Vector{Compat.UTF8String}}()
-    term_names = Vector{Compat.UTF8String}[]
+    eterm_names = Dict{Tuple{Symbol,Bool}, Vector{String}}()
+    term_names = Vector{String}[]
```

Comment:
Should it be `Vector{String}()` instead?

Reply:
Nope, it's a vector of vectors of strings :)

Reply:
Could use `Vector{Vector{Strings}}()` which might be clearer.